library/src/main/kotlin/io/beyondwords/player/MediaButtonReceiver.kt

```diff
@@ -30,7 +30,7 @@ open class MediaButtonReceiver constructor(private val mediaSessionId: Int) :
             intent.putExtra(Intent.EXTRA_KEY_EVENT, KeyEvent(KeyEvent.ACTION_DOWN, keyCode))
             intent.putExtra(EXTRA_MEDIA_SESSION_ID, mediaSessionId)
             val requestCode = mediaSessionId * 1000 + keyCode // keyCode is a 3 digit number
-            val flags = if (Build.VERSION.SDK_INT >= 31) PendingIntent.FLAG_MUTABLE else 0
+            val flags = if (Build.VERSION.SDK_INT >= 31) PendingIntent.FLAG_IMMUTABLE else 0
```

Comment:
Why those flags needs to be changed?

Reply:
Because the current implementation will throw an error: `java.lang.IllegalArgumentException: io.beyondwords.example: Targeting U+ (version 34 and above) disallows creating or retrieving a PendingIntent with FLAG_MUTABLE, an implicit Intent within and without FLAG_NO_CREATE and FLAG_ALLOW_UNSAFE_IMPLICIT_INTENT for security reasons. To retrieve an already existing PendingIntent, use FLAG_NO_CREATE, however, to create a new PendingIntent with an implicit Intent use FLAG_IMMUTABLE.`

I couldn't see anything that modified the pending intent, so I simply set it to `FLAG_IMMUTABLE` (like described in https://developer.android.com/reference/android/app/PendingIntent#FLAG_MUTABLE:  FLAG_MUTABLE should only be used when some functionality relies on modifying the underlying intent, e.g. any PendingIntent that needs to be used with inline reply or bubbles.).